pkg/nuctl/command/import.go

```diff
@@ -358,11 +353,23 @@ func (i *importProjectCommandeer) importProject(projectConfig *ProjectImportConf
 		if err != nil {
 			return err
 		}
-		if err = newProject.CreateAndWait(); err != nil {
+
+		if err := newProject.CreateAndWait(&platform.CreateProjectOptions{
+			ProjectConfig:            newProject.GetConfig(),
+			SkipTransformDisplayName: importProjectOptions.skipTransformDisplayName,
+		}); err != nil {
 			return err
 		}
+
+		// reassign created instance, it holds changes made during project creation
+		importProjectOptions.projectInfo.Project = newProject.GetConfig()
```

Comment:
Function with side effects on their parameters are usually not great unless specifically named so "updateProjectConfig"
So it's like a magic side effect of `CreateAndWait`
Why not have it return the createdProject with the changes?

Reply:
improved logic
1. import if not exists
2. get project (recently imported)
3. proceed with import project resources (apigw, functions, etc ...)